src/examples/sql-server/package.json

```diff
@@ -23,7 +23,7 @@
 	},
 	"devDependencies": {
 		"@types/node": "22.13.14",
-		"graphweaver": "2.16.1",
+		"graphweaver": "workspace:*",
```

Comment:
This is the only reference to a fixed version of a package in the mono repo. I believe it leads to multiple copies of packages in node_modules, which breaks decorators and other metadata tricks